test/gui/shared/steps/steps.py

```diff
@@ -447,32 +533,40 @@ def collaboratorShouldBeListed(
 
 @When('the user waits for the files to sync')
 def step(context):
-    waitForRootFolderToSync(context)
```

Comment:
can we remove this function if we are not using it ?

Reply:
I will do the clean-up as the followup PR

Reply:
please make the clean-up in this PR